targets/mbed/source/jerry_extapi.cpp

```diff
@@ -45,39 +45,48 @@ NAME ## _handler (const jerry_api_object_t * function_obj_p __UNSED__, \
 DECLARE_HANDLER(assert)
 {
   if (args_cnt == 1
-      && args_p[0].type == JERRY_API_DATA_TYPE_BOOLEAN
-      && args_p[0].u.v_bool == true)
+      && jerry_value_is_boolean (args[0])
+      && jerry_get_boolean_value (args[0]))
   {
     printf (">> Jerry assert true\r\n");
-    return true;
+    return jerry_create_boolean (true);
   }
   printf ("ERROR: Script assertion failed\n");
   exit (JERRY_STANDALONE_EXIT_CODE_FAIL);
-  return false;
+  return jerry_create_boolean (false);
 }
 
 DECLARE_HANDLER(led)
 {
+  jerry_value_t ret_val;
+
   if (args_cnt < 2)
   {
-    return false;
+    ret_val = jerry_create_boolean (false);
+    printf ("Error: invalid arguments number!\r\n");
+  }
+
+  if (!(jerry_value_is_number(args[0])
+        && jerry_value_is_number(args[1])))
+  {
+    ret_val = jerry_create_boolean (false);
+    printf("Error: arguments must be numbers!\r\n");
```

Comment:
Just a minor style issue: missing space after `printf` (please check other occurrences too)
Otherwise I think the PR is very close to merge.